code/game/objects/effects/spawners/structure.dm

```diff
@@ -31,3 +31,14 @@ again.
 						/obj/structure/grille,
 						/obj/structure/window/reinforced/fulltile
 						)
+
+/obj/effect/spawner/structure/closet_or_box //75% closet / 25% box, intended for maintenance
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "closet_or_box_spawner"
+
+/obj/effect/spawner/structure/closet_or_box/New()
+	if(prob(25))
```

Comment:
I did so because the mapping object can be applied selectively instead of just applying to every generic closet. For example there's some generic closets in fitness that shouldn't be boxes ever. The average box per station coefficient (Avebox/station) roughly shouldn't change as a result.

:cop: